src/main/java/frc/robot/RobotContainer.java

```diff
@@ -77,7 +89,14 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
-  private void configureButtonBindings() {}
+  private void configureButtonBindings() {
```

Comment:
we also need to add the controls for the operator (`copilotController`) (which, btw, could you rename that to `operatorController`?)